metaflow/flowspec.py

```diff
@@ -386,7 +392,7 @@ def merge_artifacts(self, inputs, exclude=[], include=[]):
 
         Parameters
         ----------
-        inputs : List[Steps]
+        inputs : List[FlowSpec]
```

Comment:
technically inputs is not a list of `FlowSpec` but a list of `Input` objects that get passed to the join step

Reply:
I think it's just `Inputs` actually. Corrected this. Thanks for catching this.